stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -4075,19 +4135,12 @@ impl NakamotoChainState {
         if let Some(signer_calculation) = signer_set_calc {
             Self::write_reward_set(chainstate_tx, &new_block_id, &signer_calculation.reward_set)?;
 
-            let cycle_number = if let Some(cycle) = pox_constants.reward_cycle_of_prepare_phase(
-                first_block_height.into(),
-                chain_tip_burn_header_height.into(),
-            ) {
-                Some(cycle)
-            } else {
-                pox_constants
-                    .block_height_to_reward_cycle(
-                        first_block_height.into(),
-                        chain_tip_burn_header_height.into(),
-                    )
-                    .map(|cycle| cycle + 1)
-            };
+            let cycle_number = pox_constants
```

Comment:
It's not obvious to me that this is semantically equivalent to the code it replaced.  For safety, can you just revert it for now?

Reply:
Will do.